integration-tests/physics/test_instant_rain.py

```diff
@@ -49,7 +49,8 @@ def run_instant_rain(dirname):
 
     # Physics schemes
     # define saturation function
-    saturation = Constant(0.5)
+    VD = FunctionSpace(mesh, "DG", 1)
+    saturation = Function(VD).assign(0.5)
```

Comment:
This shouldn't be necessary - it should also work when `saturation` is not a function.

Reply:
That depends on what the error was...